### ISSUE TITLE
How do Data Augment and Normalize get tuned?

### ISSUE THREAD

**Issue by cbsudux** [keras-team/autokeras#1088]
The docs mention that data augmentation and normalization can be specified, if not they will be automatically tuned. 

`Ycan configure the ImageBlock for some high-level configurations, e.g., block_type for the type of neural network to search, normalize for whether to do data normalization, augment for whether to do data augmentation. You can also do not specify these arguments, which would leave the different choices to be tuned automatically. See the following example for detail.`

However I do not see this in the source code. How is the tuning of data augmentation and normalization being implemented?

**Comment by haifeng-jin** [comment]
In the master branch now, you can see the tuning of these two.

**Comment by cbsudux** [comment]
@haifeng-jin , Can you post a link to the script that does it? The repo is pretty big and it's hard to find out where the tuning happens.

**Comment by haifeng-jin** [comment]
https://github.com/keras-team/autokeras/blob/master/autokeras/tuners/greedy.py
@cbsudux The tuning algorithm is here. The search space is just defined in the build function in those blocks.

**Comment by cbsudux** [comment]
Thanks, it looks like you only tune to Normalize or to Augment, as booleans. Not what transform to apply.

**Comment by fengyuan1993012** [comment]
hello, I am using autokeras to train a regression model. however, I find the normalization will lead to low accuracy. I would appreciate it if you could tell me where I can abandon the normalization layer. Thanks

**Comment by haifeng-jin** [comment]
@fengyuan1993012 You can use the `AutoModel` API to abandon it.
There is an argument in the `__init__` function of `StructuredDataBlock`.
However, as long as you set enough number of trials, it should converge on a model with good performance.
If normalization is not good for the performance, it should be abandoned through the trials.

**Comment by fengyuan1993012** [comment]
Thanks, got it



At 2021-04-07 05:19:05, "Haifeng Jin" ***@***.***> wrote:

@fengyuan1993012 You can use the AutoModel API to abandon it.
There is an argument in the __init__ function of StructuredDataBlock.
However, as long as you set enough number of trials, it should converge on a model with good performance.
If normalization is not good for the performance, it should be abandoned through the trials.
